Dockerfile

```diff
@@ -57,12 +57,12 @@ COPY --from=idmap /usr/bin/newgidmap /usr/bin/newgidmap
 COPY --from=runc /usr/bin/runc /usr/bin/runc
 COPY --from=fuse-overlayfs /usr/bin/fuse-overlayfs /usr/bin/fuse-overlayfs
 COPY --from=forge /usr/bin/forge /usr/bin/forge
-RUN chmod u+s /usr/bin/newuidmap /usr/bin/newgidmap && \
-    adduser -D -u 1000 user && \
+RUN chmod u+s /usr/bin/newuidmap /usr/bin/newgidmap
+RUN adduser -D -u 1000 user && \
```

Comment:
@steved is this just a ✂️ or was it causing an issue?

Reply:
Oh ... no issue, just that they didn't need to be tied together in one command?